dao/application_authentication_dao_test.go

```diff
@@ -309,3 +309,60 @@ func TestApplicationAuthenticationListUserOwnership(t *testing.T) {
 
 	DropSchema(schema)
 }
+
+func TestApplicationAuthenticationGetUserOwnership(t *testing.T) {
+	testutils.SkipIfNotRunningIntegrationTests(t)
+	testutils.SkipIfNotSecretStoreDatabase(t)
+	schema := "user_ownership"
+	SwitchSchema(schema)
+
+	err := TestSuiteForSourceWithOwnership(func(suiteData *SourceOwnershipDataTestSuite) error {
+		/*
+		   Test 1 - UserA tries to GET userA's application - expected result: success
+		*/
```

Comment:
there should probably be
userA's application authentication
(same for test 2 and 3)